src/Surfnet/StepupRa/RaBundle/Command/SearchRaCandidatesCommand.php

```diff
@@ -22,6 +22,23 @@
 
 class SearchRaCandidatesCommand
 {
+    /**
+     * @Assert\NotBlank(message="ra.search_ra_candidates.actor.blank")
+     * @Assert\Type("string", message="ra.search_ra_candidates.blank.type")
+     *
+     * @var string
+     */
+    public $actorId;
+
+
+    /**
+     * @Assert\NotBlank(message="ra.search_ra_candidates.institution.blank")
```

Comment:
The message is potentially confusing. For not the institution is blank but the actorInstitution is.

Reply:
Fixed